### DISCUSSION TITLE
How to know what version of Bacula are we using

### DISCUSSION THREAD

**Discussion by Harimzial** [bacularis/bacularis-app#17] [Q&A] [answered]
Title. I am quite new in system administration via open source solutions, and Linux environment. 
My company is currently using Bacula, but a version way too old. I wanted to modernise that with some graphic interface etc... and i really liked this project!
I was just wondering what version of Bacula is it using ? And if it's using the same agent as Bacula itself.

Thanks in advance, and don't hesitate to correct me on anything, always learning :)

**Comment by ganiuszka** [comment, marked as answer]
Hello @IMDEA-Matteosch,

Thanks for your introduction and for describing us what you need.

Upgrading Bacula is usually a good idea. Bacularis works with Bacula versions: ``9``, ``11``, and ``13``.

If you use Bacula ``9`` here you can read about some small limitations for ``9``:

https://bacularis.app/doc/brief/troubleshooting.html#job-file-list-does-not-show-results
https://bacularis.app/doc/brief/troubleshooting.html#searching-by-filename-does-not-work-in-restore-wizard

For the Bacula Client (agent) and Bacula Director and Storage Daemon (servers) the general rule for versions is FD <= (DIR == SD). In other words File Daemon can be in version lower or the same as Director/Storage Daemon. Director and Storage Daemon have to be in the same version and they need to be updated at the same time. These rules are not Bacularis restrictions but they come from Bacula itself.

Good luck with learning Bacula :-) I think it is a nice adventure and also challenge for people who starts with Bacula.

Best regards,
Marcin Haba (gani)

**Reply by Harimzial** [reply]
Hey @ganiuszka !

Thanks a lot for the quick answer, we are tempted to use the standalone version on docker, and completely review our backup organization, so everything would be on Bacula 13.0.2. 

I'll probably pass by again for some questions, but it's indeed very interesting ! 

All the best,
Mattéo
